client/app.jsx

```diff
@@ -16,9 +17,12 @@ export default class App extends React.Component {
 
   renderPage() {
     const { route } = this.state;
-    if (route.path === '') {
+    if (route.path === '' || route.path === '#') {
       return <Home />;
     }
+    if (route.path === 'recipe') {
+      return <Recipe />;
+    }
```

Comment:
Here you could pass in the `recipeId` like we passed the `productId` in `react-hash-routing`